Staffbase/github-action-jira-release-tagging
# SP-520 add ARN webhook call to inform the Product Enablement team

add ARN webhook call to inform the Product Enablement team

### How To test

* Code check
* MERGE IT
* wait for the next git tag on one of the services, wich have the Changelog process includes as GHA workflow (see list on https://mitarbeiterapp.atlassian.net/wiki/spaces/DEV/pages/1448050856/Change+the+Changelog+-+Overview+of+relevant+JIRA+tickets)
